src/Symfony/Component/Config/Definition/Builder/NodeFinder.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Config\Definition\Builder;
+
+/**
+ * This class finds nodes in a node tree.
+ *
+ * @author Jan Schädlich <jan.schaedlich@sensiolabs.de>
+ */
+class NodeFinder
+{
+    /**
+     * Finds a node defined by $nodePath within the given $rootNode.
+     *
+     * @param string         $nodePath The path of the node to find. e.g "doctrine.orm.mappings"
+     * @param NodeDefinition $rootNode
```

Comment:
This line and the @return could be removed

Reply:
Done